### PR TITLE
Restructure the changelog

### PR DESCRIPTION
**Pull request by 1st1**

### CODE REVIEW

**Comment by 1st1** [comment]
@vpetrovykh please push the 1.1 changes directly to this branch.

-------

Here's how this looks:


Screen Shot 2022-02-23 at 5 08 06 PM

**Comment by elprans** [comment]
This unfortunately breaks links we inserted into release tags, but I guess it isn't a big deal?

**Comment by 1st1** [comment]
We can just keep the files where they were, so the urls will keep working.

**Comment by vpetrovykh** [comment]
I was adding a few of the PRs submitted by the community in the changelog.
Also I think I may have included some changes that actually made it into 1.0 here as I wasn't sure that omitting them altogether makes sense.

So this one is a bit longer than I expected.

**Review by elprans — APPROVED** [review]

File: docs/changelog/1_x.rst [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  link (:eql:gh:`#3536`).
+* Disable changing the concrete base of a :ref:`scalar type
+  <ref_datamodel_scalar_types>` (:eql:gh:`#3529`).
+* Avoid generating pointless self joins (:eql:gh:`#2567`).
+* Fix IPv6 address parsing (:eql:gh:`#3454`).
+* If a type has object instances, it cannot be made ``abstract``
+  (:eql:gh:`#3399`).
+* Fix an issue that sometimes caused :eql:kw:`with` block variables to
+  be unusable (:eql:gh:`#3385`).
+* The ``start`` keyword is now reserved (:eql:gh:`#3396`).
```
Comment (elprans) [inline docs/changelog/1_x.rst:50]: This bit is unnecessary.  We re-reserved `start` in 1.0

**Pull request merged**